x-pack/test/api_integration/apis/transform/transforms.ts

```diff
@@ -0,0 +1,137 @@
+
+        const transform1 = body.transforms[0];
+        expect(transform1.id).to.eql(expected.apiTransformTransforms.transform1.id);
+        expect(transform1.dest.index).to.eql(expected.apiTransformTransforms.transform1.destIndex);
+        expect(typeof transform1.version).to.eql(expected.apiTransformTransforms.typeOfVersion);
+        expect(typeof transform1.create_time).to.eql(
+          expected.apiTransformTransforms.typeOfCreateTime
+        );
+
+        const transform2 = body.transforms[1];
+        expect(transform2.id).to.eql(expected.apiTransformTransforms.transform2.id);
+        expect(transform2.dest.index).to.eql(expected.apiTransformTransforms.transform2.destIndex);
+        expect(typeof transform2.version).to.eql(expected.apiTransformTransforms.typeOfVersion);
+        expect(typeof transform2.create_time).to.eql(
+          expected.apiTransformTransforms.typeOfCreateTime
+        );
+      });
+
+      it('should return 200 for transform view-only user', async () => {
+        await supertest
+          .get(`/api/transform/transforms`)
+          .auth(
+            USER.TRANSFORM_VIEWER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_VIEWER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send()
+          .expect(200);
+      });
+    });
+
+    describe('/transforms/{transformId}', function () {
+      it('should return a specific transform configuration', async () => {
+        const { body } = await supertest
+          .get('/api/transform/transforms/the-transform-1')
+          .auth(
+            USER.TRANSFORM_POWERUSER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_POWERUSER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send()
+          .expect(200);
+
+        expect(body.count).to.eql(expected.apiTransformTransformsTransformId.count);
+        expect(body.transforms).to.have.length(expected.apiTransformTransformsTransformId.count);
+
+        const transform1 = body.transforms[0];
+        expect(transform1.id).to.eql(expected.apiTransformTransformsTransformId.transform1.id);
+        expect(transform1.dest.index).to.eql(
+          expected.apiTransformTransformsTransformId.transform1.destIndex
+        );
+        expect(typeof transform1.version).to.eql(
+          expected.apiTransformTransformsTransformId.typeOfVersion
+        );
+        expect(typeof transform1.create_time).to.eql(
+          expected.apiTransformTransformsTransformId.typeOfCreateTime
+        );
+      });
+
+      it('should return 200 for transform view-only user', async () => {
+        await supertest
+          .get(`/api/transform/transforms/the-transform-1`)
+          .auth(
+            USER.TRANSFORM_VIEWER,
+            transform.securityCommon.getPasswordForUser(USER.TRANSFORM_VIEWER)
+          )
+          .set(COMMON_REQUEST_HEADERS)
+          .send()
+          .expect(200);
```

Comment:
We can probably add the same check as we did for TRANSFORM_POWERUSER here

Reply:
I agree, see comment in the test above.

Reply:
Done in 28ad1f2.